finality-provider/cmd/fpd/daemon/tx.go

```diff
@@ -28,7 +27,6 @@ func CommandTxs() *cobra.Command {
 	cmd.AddCommand(
 		authcli.GetSignCommand(),
 		btcstakingcli.NewCreateFinalityProviderCmd(),
-		finalitycli.NewUnjailFinalityProviderCmd(),
```

Comment:
did you remove this tx to be able to run an update on the local status after the tx is run onchain?

Reply:
Exactly